sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -541,7 +542,7 @@ class StreamingInnerJoinSuite extends StreamingJoinSuite {
         Utils.createDirectory(tempDir.getAbsolutePath, Random.nextFloat().toString).toString
       val stateInfo = StatefulOperatorStateInfo(path, queryId, opId, 0L, 5)
 
-      implicit val sqlContext = spark.sqlContext
+      implicit val sqlContext: SQLContext = spark.sqlContext
       val coordinatorRef = sqlContext.streams.stateStoreCoordinator
```

Comment:
how about remove line 545 and change line 546 to 
```scala
val coordinatorRef = spark.streams.stateStoreCoordinator
```

Reply:
well, maybe. For local variable, we don't need to explicit type, I revert some change. Under the this case, just remove line 545, it's ok. what do you think?

Reply:
fine to me. I see the latest commit is wip. Please ping me again after you complete this round of modifications